common/app/views/fragments/meta/metaInline.scala.html

```diff
@@ -21,6 +21,14 @@
         }
     }
 
+    @if(item.content.isPaidContent) {
```

Comment:
Just make sure the item is also an immersive and you should be good to go:
`@if(item.content.isPaidContent && item.content.isImmersive)`